commands/sandbox/create.ts

```diff
@@ -56,7 +89,7 @@ exports.handler = async options => {
           HUBSPOT_ACCOUNT_TYPE_STRINGS[
             HUBSPOT_ACCOUNT_TYPES[accountConfig.accountType]
           ],
-        accountName: accountConfig.name,
+        accountName: getAccountIdentifier(accountConfig) || '',
```

Comment:
I think you probably want to keep this as name - getAccountIdentifier returns the accountId or portalId